db/postgreSql/index.js

```diff
@@ -0,0 +1,55 @@
+const {
+  Pool,
```

Comment:
this is harder to read on different lines, I'd put it all on one

Reply:
@jaweesner It was esLint's suggestion :)